modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/display/context/WikiConfigurationDisplayContext.java

```diff
@@ -30,7 +46,113 @@ public WikiConfigurationDisplayContext(
 			WebKeys.THEME_DISPLAY);
 	}
 
+	public String getBackURL() {
+		RequestBackedPortletURLFactory requestBackedPortletURLFactory =
+			RequestBackedPortletURLFactoryUtil.create(_httpServletRequest);
+
+		return PortletURLBuilder.create(
+			requestBackedPortletURLFactory.createControlPanelRenderURL(
+				WikiPortletKeys.WIKI_ADMIN, _themeDisplay.getScopeGroup(), 0, 0)
+		).buildString();
+	}
+
+	public String getNavigation() {
+		if (Validator.isNotNull(_navigation)) {
+			return _navigation;
+		}
+
+		_navigation = ParamUtil.getString(
+			_renderRequest, "navigation", "email-from");
+
+		if (PortalUtil.isRSSFeedsEnabled()) {
+			_navigation = ParamUtil.getString(
+				_renderRequest, "navigation", "rss");
+		}
```

Comment:
Hi, @marcogalluzzi ! 😄 
Thanks for your review! Yes, the Definition of Terms in the email from is missing due to a last minute Design decision and it should be in the Figma but they haven't changed it, sorry for the misunderstanding.  And about the `ParamUtil.getString()`, I think that the first option will be rejected by Brian due to the else (he tend to avoid the else use if there is a way) and I use [the rule of three](https://dev.to/jpswade/rule-of-three-1b9d) to simplify the code, but for this I don't think that this consume enough resources, but let me think what do you think and once the frontend part has been reviewed I'll do the changes.

Reply:
Hi @BarbaraCabrera thanks for detailed explanation. This change is optional. I know the resource consumption is low but I just try to minimize it wherever I can, with the LXC environment this is getting more important, IMHO. 
BTW, I think first option can be accepted because it is not just a simple initialization.

Reply:
Cool, thanks! I think that I'm gonna leave it like this by now and when I finish the story I can take a look at it again and change it everywhere we are applying this 🙏🏼